packages/components/src/KaizenProvider/KaizenProvider.spec.tsx

```diff
@@ -9,6 +9,12 @@ const KaizenProviderWrapper = ({
   <KaizenProvider {...restProps}>{children}</KaizenProvider>
 )
 
+jest.mock("./OptionalIntlProvider", () => ({
```

Comment:
Only need to do this because RTL doesn't do shallow renders ie. it renders everything down to the last component in the tree, so in this case it is trying to render the `StaticIntlProvider` from `i18n-react-intl`